package.json

```diff
@@ -55,6 +55,7 @@
     "request": "2.79.0",
     "semver": "5.3.0",
     "socket.io": "1.7.2",
+    "socket.io-client": "1.7.2",
```

Comment:
`socket.io` requires `socket.io-client` on it's own as dependency, so there is no extra download or anything.